slices/grep.yaml

```diff
@@ -4,15 +4,16 @@ slices:
   bins:
     essential:
       - libc6_libs
-      - libpcre3_libs
+      - libpcre2-8-0_libs
     contents:
       /bin/grep:
 
   deprecated:
-    # These are shell scripts requiring a symlink from /usr/bin/bash to /bin/sh
+    # These are shell scripts requiring a symlink from /usr/bin/dash to /bin/sh
     # See: https://manpages.ubuntu.com/manpages/jammy/en/man1/grep.1.html
     essential:
-      - bash_bins
+      - dash_bins
+      - libpcre2-8-0_libs
```

Comment:
I think you misunderstood a previous review comment and changed this to libpcre2-8-0, but in 22.04, grep depends on libpcre3 -> https://packages.ubuntu.com/jammy/grep

Reply:
Yep I sure did! I think it should be good now.